Models/Reservation.cs

```diff
@@ -6,10 +6,10 @@ namespace dreamHotel.Models
 {
     public class Reservation
     {
-        public int Id { get; set; }
+        public string Id { get; set; }
```

Comment:
In a model we are using primary key as interger. I told you about only UserIds, which are string.